### DISCUSSION TITLE
希望在快捷键拓展中添加字幕的开关。

### DISCUSSION THREAD

**Discussion by Godwin9527** [the1812/Bilibili-Evolved#4851] [功能建议 / Ideas]
希望在快捷键拓展中添加字幕的开关。

**Comment by Geeyun-JY3** [comment]
不是已经有了吗：https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.6.1
> - 新增插件 `快捷键扩展 - 开关 CC 字幕`, 默认为 <kbd>Shift</kbd> + <kbd>C</kbd>. (#3407)
